ProcessMaker/Models/ProcessRequestToken.php

```diff
@@ -136,7 +136,7 @@ class ProcessRequestToken extends ProcessMakerModel implements TokenInterface
      */
     protected $appends = [
         'advanceStatus',
-        'elementDestination'
+        'elementDestination',
```

Comment:
Do not use appends since it will execute the`getElementDestinationAttribute` for each row of the tokens causing performance problems.